_translations/en.yml

```diff
@@ -54,9 +54,9 @@ en:
       special limitations on the minimum or maximum amount you can send.
 
     pci: "No PCI compliance required"
-    pcitext: "Accepting credit cards online typically requires extensive security checks in order to comply with the PCI standard. Bitcoin still requires you to <a href=\"#secure-your-wallet#\">secure your wallet</a> and your payment requests. However, you do not carry the costs and responsibilities that come with processing sensitive information from your customers like credit card numbers."
+    pcitext: "Accepting credit cards online typically requires extensive security checks in order to comply with the PCI standard. Bitcoin still requires you to <a href=\"#secure-your-wallet#\">secure your wallet</a> and your payment requests. However, since Bitcoin payments do not expose sensitive information to merchants you do not carry the costs and responsibilities that come with processing credit card numbers."
     visibility: "Get some free visibility"
-    visibilitytext: "Bitcoin is an emerging market of new customers who are searching for ways to spend their bitcoins. Accepting them is a good way to get new customers and give your business some new visibility. Accepting a new payment method has often shown to be a clever practice for online businesses."
+    visibilitytext: "Bitcoin is an growing market of new customers who are searching for ways to spend their bitcoins. Accepting Bitcoin is a good way to attract new customers and give your business some new visibility. Accepting a new payment method has often shown to be a clever practice for online businesses."
```

Comment:
Should be: `a growing` instead of `an growing` - I think it's fine to leave as it was, though ("an emerging market"). The changes in the second sentence LGTM.